App.js

```diff
@@ -1,76 +1,247 @@
 import "./styles.css";
+import React from "react";
 
-function MainContainer() {
-  return (
-    <div>
-      <colgroup className="MainContainer">
-        <SideBar />
-        <NewsFeedCol />
-        <TrendingCol />
-      </colgroup>
-    </div>
-  );
-}
+export default class FetchRandomUser extends React.Component {
```

Comment:
The name of a React component should be describing what it renders, not what actions (sometimes called "side effects," like fetching a list of users) it performs. It's a "code smell" of the component has a verb in place of a noun.

You can name this "UserFeed" or something to that effect